inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -538,6 +538,186 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
+            if (step == -1)
+                step = img_H / static_cast<float>(data_H);
+            else
+                step *= img_H;
+            res->params["step"] = Builder::asString(step);
+
+            auto min_size = details::split(res->params["min_size"], ",");
+            for (auto &size : min_size) {
+                size = Builder::asString(std::stof(size) * img_H);
+            }
+            res->params["min_size"] = details::joinVec(min_size);
+        }
+        return res;
+    });
+
+    addSpecificCreator({"PriorBoxClusteredIE"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "PriorBoxClustered",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<CNNLayer>(attrs);
+        res->params = params;
+        res->params["clip"] =
+            res->getBoolStrParamAsIntStr("clip");
+
+        auto step_h = std::stof(res->params["step_h"]);
+        auto step_w = std::stof(res->params["step_w"]);
+        if (std::abs(step_h - step_w) < 1e-5) {
+            res->params["step"] = res->params["step_w"];
+        }
+        return res;
+    });
+
+    addSpecificCreator({"ProposalIE"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Proposal",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<CNNLayer>(attrs);
+        res->params = params;
+        res->params["clip_before_nms"] =
+            res->getBoolStrParamAsIntStr("clip_before_nms");
+        res->params["clip_after_nms"] =
+            res->getBoolStrParamAsIntStr("clip_after_nms");
+        res->params["normalize"] = res->getBoolStrParamAsIntStr("normalize");
+        return res;
+    });
+
+    addSpecificCreator({"Relu"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "ReLU",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<ReLULayer>(attrs);
+        res->params = params;
+        return res;
+    });
+
+    addSpecificCreator({"Reshape"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Reshape",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<ReshapeLayer>(attrs);
+        return res;
+    });
```

Comment:
Do we really need these 2 creators? The same comment for Softmax

Reply:
We need to have these creators, because we create specific CNNLayers (Reshape layer in this case).